src/App.jsx

```diff
@@ -14,6 +15,10 @@ function App() {
         <Route path="/" element={<Home />}></Route>
         <Route path="/articles" element={<ArticlesList />}></Route>
         <Route path="/articles/:article_id" element={<Article />}></Route>
+        <Route
+          path="/articles/:article_id/comments"
```

Comment:
Do you need a path to a page of comments? If you are already rendering the Article component on the route /articles/:article_id you don't need another path.